proto/internalapi/scanner/v4/vulnerability_report.proto

```diff
@@ -48,6 +49,7 @@ message VulnerabilityReport {
     }
     string id = 1;
     string name = 2;
+    string advisory = 15;
```

Comment:
Should we consider a list, or not something we will ever do? I mean, for the future? Think of CVSS, do we want to classify the "advisory source"? This would go on the lines of giving Central the details, and let it choose which one to store.

Reply:
These things may be true, but we can always add those fields later, when needed. The proto is backwards compatible (unless we mess something up), so I think this is ok as-is